frontend/src/components/input-output/configure-ds/ConfigureDs.jsx

```diff
@@ -232,6 +238,17 @@ function ConfigureDs({
       });
   };
 
+  const updateSession = (type) => {
```

Comment:
@mohamed-siddhiq Should we move this session value update to hooks based ? Currently we are using hooks to store session related contents via hooks. @tahierhussain Thoughts ?

Reply:
@athul-rs Looks like @mohamed-siddhiq is updating the adapter related details in the session store only. In line 248, we're calling `updateSessionDetails`, which is a function to update the session store.

@mohamed-siddhiq Please correct me if I'm wrong.

Reply:
Yes @tahierhussain , you are right..